app/design/frontend/Magento/luma/Magento_SendFriend/web/css/source/_module.less

```diff
@@ -44,3 +44,24 @@
         }
     }
 }
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) { 
+    .form.send.friend {
+        .fieldset {
+            .field {
+                .control {
+                    width: 100%;
+                }  
+            }
+            padding-bottom: 10px;
+        } 
+        .action {
+            &.remove {
+                right: 0;
+                margin-left: 0;
+                text-align: right;
+                width: 65px;
```

Comment:
width should not be hardcoded as it could be longer/shorter depending on language

Reply:
Ok